### ISSUE TITLE
Logout navigation landmark tough to find for screen reader users

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/gc-digital-talent#7275]
## 🕵️‍♀️Details 

**AT Configuration** 

VoiceOver with Safari on Mac

**Tester Feedback**

I was unable to locate a link to logout of the website after entering profile data. 

**Related WCAG Criteria**

1.3.1 Info and relationships: Information, [structure](https://w3c.github.io/wcag/understanding/info-and-relationships#dfn-structure), and [relationships](https://w3c.github.io/wcag/understanding/info-and-relationships#dfn-relationships) conveyed through [presentation](https://w3c.github.io/wcag/understanding/info-and-relationships#dfn-presentation) can be [programmatically determined](https://w3c.github.io/wcag/understanding/info-and-relationships#dfn-programmatically-determined) or are available in text.

## 🧪 Testing Steps

1. Navigate to the "Review your profile" step using VoiceOver.
2. Adjust your language or DEI information. 
3. Save and quit for now.
4. Logout 

## 🌟 Potential solutions
The logout link is within a landmark, but it's poorly named. It's just called "utility". We should name it better.

## Acceptance Criteria
- [ ] Change the landmark name for login and register (and other things in there) to "Account menu", FR: "Menu du compte"
- [ ] landmark looks the same in IAP, admin, and GCDT (main)
